packages/accordions/examples/basic.md

```diff
@@ -0,0 +1,247 @@
+    content: `Pea black-eyed gumbo fennel quandong avocado greens collard celtuce bean green jícama
+    tigernut mustard bean fava corn bean azuki okra radish Kohlrabi okra. asparagus avocado lettuce
+    bean azuki horseradish pea chard. Sprout brussels squash carrot horseradish beetroot kombu seed
+    wattle Grape nori. Greens collard seed wattle bitterleaf celery gram garlic catsear taro
+    salsify potato. Pea earthnut chicory chard swiss quandong celery.`
+  }
+];
+
+const BasicExample = () => {
+  const [activeIndex, setActiveIndex] = React.useState(0);
+  const [isLongLabel, setIsLongLabel] = React.useState(false);
+  const [isHorizontal, setIsHorizontal] = React.useState(false);
+  const [isCustomIcon, setIsCustomIcon] = React.useState(false);
+  const [numberOfSteps, setNumberOfSteps] = React.useState(5);
+  const steps = allSteps.filter((step, index) => index < numberOfSteps);
+
+  return (
+    <Grid>
+      <Well isRecessed>
+        <Row>
+          <Col>
+            <Field className="u-mt-xs">
+              <Toggle checked={isHorizontal} onChange={event => setIsHorizontal(!isHorizontal)}>
+                <Label>Horizontal layout</Label>
+              </Toggle>
+            </Field>
+            <Field className="u-mt">
+              <Toggle checked={isLongLabel} onChange={event => setIsLongLabel(!isLongLabel)}>
+                <Label>Long labels</Label>
+              </Toggle>
+            </Field>
+            <Field className="u-mt">
+              <Toggle checked={isCustomIcon} onChange={event => setIsCustomIcon(!isCustomIcon)}>
+                <Label>Custom icons</Label>
+              </Toggle>
+            </Field>
+          </Col>
+          <Col>
+            <Dropdown
+              selectedItem={numberOfSteps}
+              onSelect={steps => {
+                setNumberOfSteps(Number(steps));
+                setActiveIndex(0);
+              }}
+            >
+              <SelectField>
+                <SelectLabel>Number of Steps</SelectLabel>
+                <Select isCompact>{numberOfSteps}</Select>
+              </SelectField>
+              <Menu isCompact>
+                <Item value="3">3</Item>
+                <Item value="5">5</Item>
+                <Item value="10">10</Item>
+              </Menu>
+            </Dropdown>
+            <Field className="u-mt">
+              <Label>
+                {activeIndex === numberOfSteps
+                  ? 'Steps Completed'
+                  : `Active Step: ${activeIndex + 1}`}
+              </Label>
+              <Range
+                value={activeIndex}
+                max={numberOfSteps}
+                onChange={event => {
+                  const nextActiveIndex = parseInt(event.target.value, 10);
+                  setActiveIndex(nextActiveIndex);
+                }}
+              />
+            </Field>
+          </Col>
+        </Row>
+      </Well>
+      <Row className="u-mt">
+        <Col>
```

Comment:
There are many alignment, sizing, and word-wrap issues that need to be addressed when the stepper is forced to render under extreme widths (i.e. 100px).

Reply:
I've updated the sizing and word-wrap for `Stepper`'s that are rendered in a constrained width. 

@ginnywood could you take a look at `Stepper`'s in constrained (small) widths and see if it needs further adjustments?  One way to view a constrained width in the example is navigate to the "Edit Code" section to set a `md` prop on the `<Col>` that wraps the `<Stepper>`:

```jsx
<Col md={3}>
  <Stepper>...</Stepper>
</Col>
```

We can also go through this together over video call 👍 .